policies/codingstyle-updates.txt

```diff
@@ -0,0 +1,68 @@
+A summary of the discussion thread so far.  Not surprisingly, it's all about
+the whitespace. :)
+
+The descriptions here were written to be understandable stand-alone.  Once we
+come to a conclusion, we'll wordsmith them into the coding style.
+
+Do not put a space after sizeof; do use parens:
+
+	sizeof(foo)
+
+When breaking long lines, if there are Boolean conditionals, put them at the
+start of the line.  Consider doing this consistently, and don't merge even if
+they fit.  Here is a bad example, it should be three lines:
+
+	some_long_condition()
+	&& short1() && short2()
+
+Related conditions should be on one line if they fit, else given an extra
+level of indent
+
+	some_long_condition()
+	    && (short1() || short2())
+	    && some_other_flag != 0
+
+If the expression for an if statement does not fit, indent the continuation
+lines an extra level
+
+	if (this_is_true()
+		&& !that_is_false()) {
+	    code();
+	    ...
+
+Try not to break long lines within a function call, but if you have to, indent
+the rest of the parameter list to be after the function's opening paren.
+
+Remember a blank line after variable declarations (even local ones).
+
+Treat a single-statement with comment as if it were multi-line and use curly
+braces. Note that this could end up having "cascading curly" effects.
+
+	if (test()) {
+	    /* already alerted */
+	    close();
+	}

```

Comment:
I wonder what would you think of
```
if (test()) /* alredy alerted? */
    close();

Reply:
My personal view is that the comment should go on the line above the if, or below it.
```
/* If already alerted, go away */
if (test())
    close();
```

Reply:
Just in case for reference, "you" in my remark was plural :-) Otherwise as for placing it above, it works for single if, but imagine you have say three-term elsif chain...

Reply:
it's ugly but i added some words.